frameworks/cassandra/tests/test_jmx.py

```diff
@@ -0,0 +1,261 @@
+            "$JAVA_HOME/bin/java "
+            "-Duser.home=$MESOS_SANDBOX "
+            "-Djdk.tls.client.protocols=TLSv1.2 "
+            "-Djavax.net.ssl.trustStore={trust_store} "
+            "-Djavax.net.ssl.trustStorePassword={trust_store_password} "
+            "-Djavax.net.ssl.keyStore=$MESOS_SANDBOX/jmx/key_store -Djavax.net.ssl.keyStorePassword=deleteme "
+            "-Djavax.net.ssl.trustStoreType=JKS -Djavax.net.ssl.keyStoreType=JKS -jar jmxterm-1.0.1-uber.jar "
+            "-l service:jmx:rmi:///jndi/rmi://$MESOS_CONTAINER_IP:7199/jmxrmi -u admin -p adminpassword "
+            "-s -v silent -n".format(
+                trust_store=trust_store, trust_store_password=trust_store_password
+            )
+        )
+
+        input_jmx_commands = " < jmx_beans_command.txt"
+
+        full_cmd = "bash -c '{}{}'".format(cmd, input_jmx_commands)
+
+        _, output, _ = sdk_cmd.run_cli(
+            "task exec {} {}".format(node_task_id_0, full_cmd), print_output=True
+        )
+
+        assert "org.apache.cassandra.net:type=FailureDetector" in output
+        assert "org.apache.cassandra.net:type=Gossiper" in output
+
+        input_jmx_commands = " < jmx_domains_command.txt"
+        full_cmd = "bash -c '{}{}'".format(cmd, input_jmx_commands)
+        rc, output, stderr = sdk_cmd.run_cli(
+            "task exec {} {}".format(node_task_id_0, full_cmd), print_output=True
+        )
+
+        assert "org.apache.cassandra.metrics" in output
+        assert "org.apache.cassandra.service" in output
+
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, foldered_name)
+        uninstall_jmx_secrets()
+        for job in test_jobs:
+            sdk_jobs.remove_job(job)
+
+
+def random_string(length=10):
+    letters = string.ascii_lowercase
+    return "".join(random.choice(letters) for i in range(length))
+
+
+def write_to_file(content, file_path):
+    text_file = open(file_path, "w+")
+    text_file.write(content)
+    text_file.close()
+
+
+def generate_jmx_command_files(task_id: string):
+    cmd = "\n".join(
+        ["echo beans >> jmx_beans_command.txt && ", "echo domains >> jmx_domains_command.txt"]
+    )
+    full_cmd = "bash -c '{}'".format(cmd)
+    rc, stdout, stderr = sdk_cmd.run_cli(
+        "task exec {} {}".format(task_id, full_cmd), print_output=True
+    )
+    assert rc == 0, "Error creating jmx_commands file"
+
+
+def install_jmxterm(task_id: string):
+    jmx_term_url = "https://downloads.mesosphere.io/jmx/assets/jmxterm-1.0.1-uber.jar"
+    cmd = "/opt/mesosphere/bin/curl {} --output jmxterm-1.0.1-uber.jar".format(jmx_term_url)
+    full_cmd = "bash -c '{}'".format(cmd)
+    rc, stdout, stderr = sdk_cmd.run_cli(
+        "task exec {} {}".format(task_id, full_cmd), print_output=True
+    )
+    assert rc == 0, "Error downloading jmxterm {}".format(jmx_term_url)
+
+
+def create_secret(secret_value: str, secret_path: str) -> None:
+
+    delete_secret(secret=secret_path)
```

Comment:
Replace with the sdk_security cmd as this extra method doesn't seem to do anything different